Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/measure.h

```diff
@@ -211,6 +211,54 @@ squared_edge_length(typename boost::graph_traits<PolygonMesh>::edge_descriptor e
   return squared_edge_length(halfedge(e, pmesh), pmesh, np);
 }
 
+/**
+  * \ingroup PMP_measure_grp
+  *
+  * Finds the longest edge of a given polygon mesh.
+  *
+  * @tparam PolygonMesh a model of `HalfedgeGraph`
+  * @tparam NamedParameters a sequence of \ref bgl_namedparameters "Named Parameters"
+  *
+  * @param pmesh the polygon mesh to which `h` belongs
```

Comment:
there is no `h` mentioned above. Copy-paste issue?

Reply:
Indeed. Thank you for the correction.